tutorials/btp-cockpit-entitlements/btp-cockpit-entitlements.md

```diff
@@ -1,77 +1,77 @@
-
-Every service has its own available plans:
-
-1. Go to [SAP Discovery Center](https://discovery-center.cloud.sap/) to obtain information on which plans are available. To do so, search for the name of the product you are interested in.
-
-2. Select the tile of the product you are interested in.
-
-3. From the available tabs, select **Pricing**. You can now see all Service Plans available (such as **Free**, **Default** / **Standard** or **Trial** service plans). As an example, see the screenshot below.
-
-!![Discovery Center](tutorial-instance-4-discovery-center.png)
-
----
-
-[ACCORDION-BEGIN [Step 1: ](Understanding entitlements and quota)]
-Before you get started, you should understand the difference between entitlements and quota, because you'll encounter these two terms quite often.
-
-- **Entitlements** are your right to provision and consume a resource. They refer to **what** you're entitled to use (e.g., which services and service plans).
-
-- **Quota** is the numeric quantity that defines the maximum allowed consumption of that resource. In other words, **how much** of a service plan you're entitled to use.
-
-Entitlements and quota are managed at the global account and directory levels, distributed to directories and subaccounts, and consumed by the subaccounts.
-
-
-[VALIDATE_1]
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 2: ](Assign entitlements)]
-1. In your global account, navigate to **Entitlements** > **Entity Assignments**.
-2. In the **Select Entities** selection, choose the directory or subaccount that you want to assign entitlements to.
-3. Click **Go**.
-
-    You'll see a table for the directory/ subaccount with all entitlements it already has.
-4. Click **Configure Entitlements**, then select **Add Service Plans**.
-
-    A popover appears with all entitlements available for your directory/ subaccount.
-
-    !![Configure Subaccount Entitlements](subaccounts.gif)
-
-5. Select the service and then the plan you want to entitle.
-
-    > If you are in a global account using the consumption-based commercial model, you can select the service plan called **Free** here to test out the service without any costs.
-
-6. Select the amount of quota you need.
-7. Click **Add Service Plans** to exit the popover.
-8. Click **Save**.
-
-  You have now successfully configured entitlements.
-
-[DONE]
-[ACCORDION-END]
-
-
----
+---
+title: Manage Entitlements Using the Cockpit
+description: Configure entitlements and quota using the cockpit. (Free Tier ready)
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, topic>cloud, software-product-function>sap-btp-cockpit, tutorial>free-tier]
+primary_tag: products>sap-business-technology-platform

```

Comment:
Please correct tag (products -> software-product)

Reply:
Hey @MichaelCzcz  - sorry, I do not own these tutorials I just adapted some steps in reference to the documentation. Can you share more details what's wrong with the tags?
Annika